zipkin-ui/js/component_data/default.js

```diff
@@ -21,7 +21,7 @@ export default component(function DefaultData() {
     const query = convertToApiQuery(window.location.search);
     const serviceName = query.serviceName;
     if (serviceName) {
-      const apiURL = `/api/v1/traces?${queryString.stringify(query)}`;
+      const apiURL = `/zipkin/api/v1/traces?${queryString.stringify(query)}`;
```

Comment:
Sorry. I made a mistake with the creation of the pull request. I made a mistake with the choice of destination.  
If you do not mind, I will close this pull request.